package.json

```diff
@@ -5,7 +5,9 @@
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "serve": "nodemon server/index.js"
+    "serve": "nodemon server/index.js",
+    "client:dev": "webpack-dev-server --mode development",
+    "start": "node server/index.js"
```

Comment:
https://www.npmjs.com/package/concurrently is a nice package that allows you to create another script that will run both of these

Reply:
Yeah, that was a consideration. However, I decided against it, since I wanted to have separate logging for frontend and backend. It wasn't too much of a hassle either, since I'm using VSCode's integrated CLIs for development.